src/components/Signin.jsx

```diff
@@ -0,0 +1,281 @@
+                                {status === "ERROR" && (
+                                    <h5 className="text-danger">Oops! There was an error.</h5>
+                                )}
+                            </div> */}
+                            <div className="form-group text-center">
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignIn
+                              </Button>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                                width="60%"
+                                marginLeft="5px"
+                              >
+                                Forgot your password?
+                              </Button>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignUp
+                              </Button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <Seperator >
+                                <h2><span>OR</span></h2>
+                              </Seperator>
+                              <DisplayForLinks >
+                                  <li>
+                                    <AuthLink  
+                                      color ="white" bgColor="#3B5998" 
+                                      href="#" className="btn"
+                                    >
+                                      <i className="fa fa-facebook fa-fw"></i> Signin with Facebook
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      bgColor="black" color="white" 
+                                      hoverBgColor="darkgrey" href="#" 
+                                      className=" btn"
+                                    >
+                                      <i className="fa fa-github "></i> Signin with Github
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
```

Comment:
Inspect your page and remove unnecessary properties. 
![pr_review](https://user-images.githubusercontent.com/36301481/91658803-16dbbb00-eae9-11ea-8117-3a9bedd304b5.png)

Reply:
float :left is required, otherwise the icon will not be on the side , but it will be on the top of the content.

Reply:
Read the comments mentioned below.

Reply:
i read them, you didnt understand my problem, here when i remove the float propery which i set it to left, the icon will be on top of the content "Signin with linkedin" , but we want it to be beside of it, so thatsy, i have set the float property.Now did you understand?